src/viz-parole-population/VizParolePopulation.js

```diff
@@ -1,26 +1,128 @@
 import PropTypes from "prop-types";
 import React from "react";
+import { group, sum } from "d3-array";
 import styled from "styled-components";
+import {
+  DIMENSION_DATA_KEYS,
+  DIMENSION_KEYS,
+  DIMENSION_LABELS,
+  DIMENSION_MAPPINGS,
+  THEME,
+} from "../constants";
+import {
+  formatAsNumber,
+  formatDemographicValue,
+  recordIsTotal,
+} from "../utils";
+import ProportionalBar from "../proportional-bar";
 import StateDistrictMap from "../state-district-map";
-import { recordIsTotal } from "../utils";
+import Statistic from "../statistic";
+
+const BAR_CHART_VISUALIZATION_COLORS = {
+  [DIMENSION_KEYS.gender]: THEME.colors.gender,
+  [DIMENSION_KEYS.age]: THEME.colors.age,
+  [DIMENSION_KEYS.race]: THEME.colors.race,
+};
 
 const VizParolePopulationContainer = styled.div`
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
 `;
+
 const VizWrapper = styled.div`
-  flex: 0 1 auto;
+  flex: 0 1 49%;
 `;
 
 const MapWrapper = styled.figure`
   margin: 0;
 `;
+
 const MapCaption = styled.figcaption`
   color: ${(props) => props.theme.colors.body};
   font: ${(props) => props.theme.fonts.body};
 `;
 
+const ParoleDemographicsWrapper = styled.div`
+  height: 100%;
+  margin: 0;
+  padding-left: 56px;
```

Comment:
One thing in particular that I'm not quite grasping in regards to how to get things looking good on smaller devices is how we'll handle styles that are needed when two side by side elements are in a non-wrapped context (i.e. on a Desktop), but then transition to a wrapped context (mobile) where the styles are no longer needed.

The specific example here is that we need some space between the state map and the demographic bar charts when they're side by side, but that space is no longer needed once the screen size is reduced and the bar charts wrap below the map.